.isort.cfg

```diff
@@ -9,7 +9,7 @@ import_heading_firstparty=cisagov Libraries
 # Should be auto-populated by seed-isort-config hook
```

Comment:
Can't this jazz be removed now that we are no longer using seed-isort-config?  At a minimum the comment just before this line should be changed.

Reply:
That's in the current cycle of skeleton PRs being pushed out. This is an old PR I hadn't had a chance to resolve last week (after the conversation with @felddy about `apb` as a script or a package).